packages/@glimmer/interfaces/lib/managers/modifier.d.ts

```diff
@@ -23,7 +23,7 @@ export interface ModifierCapabilities extends Capabilities {
 export interface ModifierManager<ModifierStateBucket> {
   capabilities: ModifierCapabilities;
   createModifier(factory: unknown, args: Arguments): ModifierStateBucket;
-  installModifier(instance: ModifierStateBucket, element: SimpleElement, args: Arguments): void;
+  installModifier(instance: ModifierStateBucket, element: Element, args: Arguments): void;
```

Comment:
FWIW, I personally consider this a non-breaking change. `SimpleElement` is a strict subset of the `Element` API.